routers/likes.py

```
@@ -1,3 +1,31 @@
+    return like_post(post_id)
+
+@router.get("/comments/{comment_id}/likes")
+def like_comment(comment_id: int):
+    return like_comment(comment_id)
+
+@router.get("/likes/me")
+def like_me():
+    return like_me()
+
+@router.post("posts/{post_id}/likes")
+def like_post(post_id: int):
```

Ruff (0.14.11)

Redefinition of unused `like_post` from line 6: `like_post` redefined here

(F811)